ATM.py

```diff
@@ -71,35 +74,38 @@ def atm_machine(id, atm_dic):
             print('action not detected')
 
     else:
-        print('there is an error i the ID you etered')
+        print('There is an error in the ID you entered')
 
 
 def main():
-    """
+    """Do the ATM operation and save the changes
 
-    :arg
-        id(int) - 9 digits ID (example: 322781145)
+    will do the desired ATM operation according to an
+    ID that has been accepted until receive -1 as an ID.
+    all the changes will be saved to the ATM.
 
-    :return:
-        will do the desired ATM operation accordig to an
-        ID that has been accepted until receive -1 as an ID.
-        all the chages will be saved to the ATM.
+    Arg:
+        id (int): 9 digits ID (example: 322781145)
 
+    Raises:
+        Exception as e: No such file or directory
+            if the path to the file not exist will be an error.
     """
-    if os.path.isfile(sys.argv[PATH]):
-        atm_dic = file_to_dic(sys.argv[PATH])
+    try:
+        atm_dic = file_to_dic(sys.argv[PATH_TO_FILE])
         id = input('please enter your ID\n')
 
         while id != '-1':
-            atm_machine(id, atm_dic)
+            operation_of_atm(id, atm_dic)
             id = input('please enter your ID\n')
 
-        with open(sys.argv[PATH],'w') as newfile:
+        with open(sys.argv[PATH], 'w') as newfile:
             for id in atm_dic:
-                newfile.writelines(f"{id} {atm_dic[id][PASSWORD]} {atm_dic[id][PASSWORD]}\n")
+                newfile.writelines(f"{id} {atm_dic[id][PASSWORD]} "
+                                   f"{atm_dic[id][BALANCE]}\n")
 
-    else:
-        print('file not found')
+    except Exception as e:
+        print(e)
```

Comment:
If you catch an exception just to print it, its redundant.
Its ok to let the program fail if there was an error.